packages/story-editor/src/app/story/utils/test/isEmptyStory.js

```diff
@@ -54,4 +54,12 @@ describe('isEmptyStory', () => {
     storyWithExtraElements[0].elements.push({});
     expect(isEmptyStory(storyWithExtraElements)).toBeFalse();
   });
+
+  it('should not count story with an attachment as empty', () => {
+    const storyWithAttachment = [createPage()];
+    const testUrl = 'https://testurl.com';
+
+    storyWithAttachment[0].pageAttachemnt.push({ url: testUrl });
+    expect(isEmptyStory(storyWithAttachment)).toBeFalse();
```

Comment:
Would be good to test that it returns `true` correctly as well, for example when `pageAttachment` is `null` (or not set) and also when it was set but then deleted, then the `pageAttachment` value would be:
```
    {
        url: '',
        icon: '',
        needsProxy: false,
      }
```

Reply:
@miina thanks for the suggestion! I've added the tests